### DISCUSSION TITLE
Feature the Quick Start repository more in docs and blog posts

### DISCUSSION THREAD

**Discussion by rohancragg** [Azure/PSRule.Rules.Azure#2313] [Ideas]
I found [the Quick Start repository](https://github.com/Azure/PSRule.Rules.Azure-quickstart) very useful, once I found it, but I don't feel that it's featured enough to make it discoverable when you first start trying to learn how to use PsRule and PsRule for Azure.

I created [this repository](https://github.com/qnrl/PsRule-Demo) after reading [this blog post](https://techcommunity.microsoft.com/t5/itops-talk-blog/psrule-introduction-to-infrastructure-as-code-iac-testing/ba-p/3580746) but the Quick Start repo is almost identical to mine and may have saved me some time if I'd discovered it sooner in my journey, but it is not sign-posted as a a place to start, other than in a link at the bottom of the blog post.

**Comment by rohancragg** [comment]
I think there should be a link to The Quick Start repo on this page
https://github.com/Azure/PSRule.Rules.Azure/blob/main/docs/about.md

**Reply by BernieWhite** [reply]
Thanks for the suggestion @rohancragg we can definitely do this.
